### ISSUE TITLE
[Suggestion] Key instead of object

### ISSUE THREAD

**Issue by SeDark** [OnlyNoobDied/GadgetsMenu#688]
Hello, lately my users complain that they can move in some moments the object that is used to activate abilities, gadgets, etc and then they are left in the inventory without being able to move it again.

I have not managed to reproduce it but I have been able to see how they have in their inventories the object, for example that of the skills of the disguises.

I come to propose that this object can be changed in all menus (optionally) by a key to activate them, thus saving problems by being a key and not an object in the inventory.

Thanks for reading and I hope this is implemented

**Comment by OnlyNoobDied** [comment]
Are they in creative mode? If yes, their is no way to fix this issue.

**Comment by SeDark** [comment]
No, the only ones with creative mode are the administrators, but equally it is a solvable bug or I would not like this functionality.

Thanks for your answer

**Comment by OnlyNoobDied** [comment]
Hope you can reproduce the issue with a video.

**Comment by OnlyNoobDied** [comment]
@SeDark

**Comment by SeDark** [comment]
https://gyazo.com/eea5b3eda29393577e7aefe493ff0517

I use a plugin called CMI, which is the axis of my server (similar to essentials), apparently they can move the objects through the guis of that plugin.

What I'm asking isn't that it be fixed (which would also be fine) but that all the objects that are given to the player in the inventory are replaced by a hotkey.

@OnlyNoobDied

**Comment by OnlyNoobDied** [comment]
That's quite hard to make it activate via Hot Key. You should ask the author of CMI to fix it's GUI issue.

**Comment by amnrmain** [comment]
This is more a suggestion since I believe that doing it would remove a lot of problems, whether current or future.

**Comment by SeruhioX** [comment]
This is more a suggestion since I believe that doing it would remove a lot of problems, whether current or future.

**Comment by OnlyNoobDied** [comment]
Can you reproduce the issue with the latest GadgetsMenu update & latest CMI build?
